config/service.go

```diff
@@ -55,6 +55,30 @@ type ServiceTelemetryLogs struct {
 	// Encoding sets the logger's encoding.
 	// Valid values are "json" and "console".
 	Encoding string `mapstructure:"encoding"`
+
+	// DisableCaller stops annotating logs with the calling function's file
+	// name and line number. By default, all logs are annotated.
+	DisableCaller bool `mapstructure:"disableCaller"`
+
+	// DisableStacktrace completely disables automatic stacktrace capturing. By
+	// default, stacktraces are captured for WarnLevel and above logs in
+	// development and ErrorLevel and above in production.
+	DisableStacktrace bool `mapstructure:"disableStacktrace"`
+
+	// OutputPaths is a list of URLs or file paths to write logging output to.
+	// See Open for details.
```

Comment:
See what? I think this comment is not clear for our users.

Reply:
The origin comments from Open is too long. I added a summary and point to the source file.